### PR TITLE
Access block explorer

### LINKED ISSUE #25: Add documentation on Block Explorers
## Issue Overview

Users can now access block explorers from our framework, kindly write a small note about this.

## Proposed Solutions or Ideas

- Your documentation should be clear and concise.
- Use images where appropriate
- Use proper styling and hierarchy for text
- Use proper styling for code and variable names.
- Use proper callouts when required

Refer to docusaurus for more info: https://docusaurus.io/

### PR DESCRIPTION
**Pull request by Benjtalkshow**
closes #25 
# Add documentation on Block Explorers

## Types of change

- [x] Enhancement

### CODE REVIEW

**Comment by Nadai2010** [comment]
Hi @Benjtalkshow, Thank you for your work

**Some ideas we can use:**

- To access the explorers, we can provide a direct list:
```bash
* [Starkscan]()
* [Voyager]()
* [StarkCompas]()
``` 
- Depending on the chain in `sepolia` or `mainnet` set in the `Scaffold.config` file, it will retrieve its respective explorers for `Sepolia` or `Mainnet`. We can mention this.

- We could also include a link to the [document listing all explorers on Starknet](https://docs.starknet.io/documentation/tools/ref_block_explorers/)

**Review by metalboyrick — CHANGES_REQUESTED** [review]
Thanks for the hard work, kindly address @Nadai2010 's comments and you should be all good!

**Comment by Benjtalkshow** [comment]
> Thanks for the hard work, kindly address @Nadai2010 's comments and you should be all good!

Alright. Thanks

**Review by metalboyrick — APPROVED** [review]
Let's wait for @Nadai2010 comments!

**Comment by Benjtalkshow** [comment]
> Let's wait for @Nadai2010 comments!

alright ser

File: static/.DS_Store
Comment (Nadai2010) [inline static/.DS_Store:1]: No commit this file

File: static/img/.DS_Store
Comment (Nadai2010) [inline static/img/.DS_Store:1]: No commit this file

**Review by Nadai2010 — CHANGES_REQUESTED** [review]

**Comment by Benjtalkshow** [comment]
@Nadai2010 
Made the requested chnages ser

File: .gitignore
Comment (Nadai2010) [inline .gitignore:1]: Remove .gitginore

**Review by Nadai2010 — CHANGES_REQUESTED** [review]

**Review by Nadai2010 — APPROVED** [review]

**Pull request merged**